freud/order.pyx

```diff
@@ -494,6 +494,173 @@ cdef class TransOrderParameter:
     def __str__(self):
         return repr(self)
 
+cdef class Steinhardt:
+    R"""Compute the local Steinhardt [Steinhardt1983]_rotationally invariant
+    :math:`Q_l` :math:`Wl` order parameter for a set of points.
+
+    Implements the local rotationally invariant :math:`Q_l` or :math:`Wl` order
+    parameter described by Steinhardt. For a particle i, we calculate the
+    average order parameter by summing the spherical harmonics between particle
+    :math:`i` and its neighbors :math:`j` in a local region:
+    :math:`\overline{Q}_{lm}(i) = \frac{1}{N_b} \displaystyle\sum_{j=1}^{N_b}
+    Y_{lm}(\theta(\vec{r}_{ij}), \phi(\vec{r}_{ij}))`. The particles included in
+    the sum are determined by the rmax argument to the constructor.
+
+    For :math:`Ql`, this is then combined in a rotationally invariant fashion to
+    remove local orientational order as follows:
+    :math:`Q_l(i)=\sqrt{\frac{4\pi}{2l+1} \displaystyle\sum_{m=-l}^{l}
+    |\overline{Q}_{lm}|^2 }`.
+
+    For :math:`W_l`, it is then defined as a weighted average over the
+    :math:`\overline{Q}_{lm}(i)` values using Wigner 3j symbols
+    (Clebsch-Gordan coefficients). The resulting combination is rotationally
+    (i.e. frame) invariant.
+
+    The :data:`average` argument in the constructor provides access to a variant
+    of this parameter that performs a average over the first and second shell
+    combined [Lechner2008]_. To compute this parameter, we perform a second
+    averaging over the first neighbor shell of the particle to implicitly
+    include information about the second neighbor shell. This averaging is
+    performed by replacing the value :math:`\overline{Q}_{lm}(i)` in the
+    original definition by the average value of :math:`\overline{Q}_{lm}(k)`
+    over all the :math:`k` neighbors of particle :math:`i` as well as itself.
+
+    The :data:`norm` constructor arguement provides
+    normalized versions of the plain :math:`Ql` or :math:`Wl` or normalized
+    average if the :data:`average` flag is set to true.  where the normalization
```

Comment:
Fix the incomplete sentence here. Maybe write "Normalization is performed by..."